### ISSUE TITLE
Live Camera recognization screen are missing

### ISSUE THREAD

**Issue by EusufUddin8926** [kuru0777/face-recognition-with-flutter#1]

**Comment by kuru0777** [comment]
> 

Can you try without upgrading the dependency version? You should try the dependencies in the pubspec.yaml file before upgrading the versions of the Google ml kit, camera and tflite packages.

**Comment by EusufUddin8926** [comment]
Yes. I am running this app with specific pubdev version, but it not working with live camera also from image to recognize accuracy too low

**Comment by kuru0777** [comment]
> Yes. I am running this app with specific pubdev version, but it not working with live camera also from image to recognize accuracy too low

Someone reached out last month and it was working then. If u want to reach me mehmetkuru0777@gmail.com
